### PR TITLE
Alzheimers Disease Prediction using Deep Neural Network

### PR DESCRIPTION
**Pull request by srajan-kiyotaka**
Deep Learning CNN Model To Predict Alzheimer's Disease Using CT Scan of Brain.
In This Model, The Model is Trained Using The CT Scan Image Dataset.
- The model accuracy is 95.63%.
- Test accuracy is also 98.68 percent
- The Model is not overfitted or underfit.
- I had also added the readme file as well as the dataset and images used.

Issue No. : #396

### CODE REVIEW

**Comment by Niketkumardheeryan** [comment]
@srajan-kiyotaka use predefined matrices for accuracy

**Comment by srajan-kiyotaka** [comment]
@Niketkumardheeryan 
I had done the changes,
e.g. I had added the accuracy matrix which calculates the test accuracy during each epoch,
while training.
